Wolox/WLXQuickBloxUtils-sample
# Conversation list view

## Sumary

**Trello card**: https://trello.com/c/UOVumCsF/13-pantalla-de-lista-de-conversaciones

Conversation List view added to the Tab Bar

A brief description of the pull request
## Test Plan
- Open the app
- Login with username 'asd@asd.com' password 'asdasdasd'
- Go to the conversations tab. If there are no conversations, return to the Users tab and begin a conversation, then return to the conversations tab and select it. If there are conversations, select one.
- Check that the conversation opens and the messages sent are there.
## Screenshots

![gif](https://cloud.githubusercontent.com/assets/5349650/8961372/3917cde8-35eb-11e5-94a2-8dcc3babb28a.gif)
